tests/flow/test_query_validation.py

```diff
@@ -573,3 +573,21 @@ def test38_return_star_union(self):
                 assert(False)
             except redis.exceptions.ResponseError as e:
                 self.env.assertContains("All sub queries in an UNION must have the same column names", str(e))
+
+    def test39_invalid_query(self):
```

Comment:
Please rename to a more appropriate name, this is a bit too generic, needs to indicate what exactly is being tested.